tests/test_rdf.py

```diff
@@ -86,3 +86,78 @@ def compare_gromacs_xvg(ref_filename, r, g_r):
     g_r0 = data[:, 1]
     eq(r, r0, decimal=2)
     eq(g_r, g_r0, decimal=2)
+
+@pytest.mark.parametrize('periodic, opt', [(True, True), (True, False), (False, True), (False, False)])
+def test_rdf_t(get_fn, periodic, opt):
+    traj = md.load(get_fn('tip3p_300K_1ATM.xtc'), top=get_fn('tip3p_300K_1ATM.pdb'))
+    pairs = traj.top.select_pairs('name O', 'name O')
+    times = list()
+    for j in range(2):
+        for i in range(4):
+            times.append([j*4+i, j*4])
+
+    r, g_r_t = md.compute_rdf_t(traj, pairs, times, r_range=(0, 1), periodic=periodic, opt=opt)
+
+    assert g_r_t.shape == (8, 200)
```

Comment:
Any other sanity checks that can be added here? Maybe i.e. assert the location of a peak

Reply:
Yep, I need to think a little more on this.